src/tests/topp/FeatureFinderMultiplex_1_output.featureXML

```diff
@@ -1,172 +1,173 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 <featureMap version="1.9" id="fm_7804704400743266335" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/OpenMS/OpenMS/develop/share/OpenMS/SCHEMAS/FeatureXML_1_9.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<UserParam type="stringList" name="spectra_data" value="[FeatureFinderMultiplex_1_input.mzML]"/>
```

Comment:
TOPP tests fail here. Does Jenkins use the absolute file path instead of the file name?